pandas/tests/indexing/multiindex/test_partial.py

```diff
@@ -178,9 +178,9 @@ def test_partial_loc_missing(self, multiindex_year_month_day_dataframe_random_da
         # assert (self.ymd.loc[2000]['A'] == 0).all()
 
         # Pretty sure the second (and maybe even the first) is already wrong.
-        with pytest.raises(Exception):
+        with tm.external_error_raised(KeyError):
```

Comment:
what are the errors?

Reply:
Just a simple `KeyError: 6` and `KeyError: (2000, 6)`

Reply:
i think its worth matching this

Reply:
done